test/abstractServer.cpp

```diff
@@ -105,7 +105,7 @@ std::string AbstractServer::Base64Decode(const std::string& in) {
             break;
         }
         // NOLINTNEXTLINE (cppcoreguidelines-avoid-magic-numbers)
-        val = (val << 6) + T[c];
+        val = ((uint)val << (uint)6) + T[c];
```

Comment:
Please don't use C-style casts
```suggestion
        val = (static_cast<uint>(val) << static_cast<uint>(6)) + T[c];
```

Reply:
Got it!